internal/commands/image/tag.go

```diff
@@ -12,8 +12,10 @@ import (
 
 func tagCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Short: "Create a new tag from an application image",
-		Use:   "tag SOURCE_APP_IMAGE[:TAG] TARGET_APP_IMAGE[:TAG]",
+		Short: "Create a new tag from an App Image",
+		Use:   "tag [SOURCE_APP_IMAGE] [TARGET_APP_IMAGE]",
```

Comment:
I'm not sure about this change, the fields without brackets means they are mandatory, the tags inside brackets that they are optional.

Reply:
The idea was to be consistent with the output of docker image ls who returns repo:tag info in an "APP IMAGE" column. I have rolled back this change, this can be discussed with the other ticket created